### PR TITLE
Javascript javascript1 week1

### PR DESCRIPTION
**Pull request by suvarna-ratna**
home work

### CODE REVIEW

File: javascript/javascript1/week1/Age-ify.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 let yearOfBirth = 1992
 let yearFuture = 2027
 let age = yearFuture-yearOfBirth
+
```
Comment (KavyaReddy732) [inline javascript/javascript1/week1/Age-ify.js:4]: you could use 'const' to declare the variables unless you want to re-assign the value of the variable.

File: javascript/javascript1/week1/Ez-Namey.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 let firstWords = ['Easy','Awesome','Corporate','Great','Cool','Adventure','Systematic','Dynamic','Logical','Practical'];
 let secondWords = ['Super', 'Technology', 'Idea','Media','Information','Launched','Problems','Solutions','Team','Support'];
+
 const randomNumber1 = Math.floor(Math.random() * firstWords.length);
 const randomNumber2 = Math.floor(Math.random() * secondWords.length);
 const startupName = firstWords[randomNumber1]+ " " + secondWords[randomNumber2];
+
```
Comment (KavyaReddy732) [inline javascript/javascript1/week1/Ez-Namey.js:7]: super nice work.

File: javascript/javascript1/week1/Ez-Namey.js [hunk 3]
```diff
@@ -1,6 +1,8 @@
 let firstWords = ['Easy','Awesome','Corporate','Great','Cool','Adventure','Systematic','Dynamic','Logical','Practical'];
 let secondWords = ['Super', 'Technology', 'Idea','Media','Information','Launched','Problems','Solutions','Team','Support'];
+
```
Comment (KavyaReddy732) [inline javascript/javascript1/week1/Ez-Namey.js:3]: same here use 'const'

File: javascript/javascript1/week1/Housey - pricey.js [hunk 4]
```diff
@@ -21,9 +21,9 @@ const costPayingByPeter = 2500000;
  //Julia house price
```
Comment (KavyaReddy732) [inline javascript/javascript1/week1/Housey - pricey.js:21]: you can use array format for volume and for the price of the house as well like: 
const volumeInMeters = [widthOfHouse[0]*heightOfHouse[0]*depthOfHouse[0], widthOfHouse[1]*heightOfHouse[1]*depthOfHouse[1]];
compare both the values using index value of an array.

File: javascript/javascript1/week1/Goodboy-Oldboy.js [hunk 5]
```diff
@@ -3,6 +3,7 @@ let dogYearFuture = 2027;
 let dogYear = dogYearFuture - dogYearOfBirth;
 //console.log(dogYear);
```
Comment (KavyaReddy732) [inline javascript/javascript1/week1/Goodboy-Oldboy.js]: remove the commented code while commit the code unless it is description or for the future understanding.

**Review by KavyaReddy732 — COMMENTED** [review]
overall, it's very nice and clean except for small modifications.

File: javascript/javascript1/week1/Age-ify.js (hunk 1 above)
Comment (suvarna-ratna) [reply javascript/javascript1/week1/Age-ify.js:4]: OK Kavya .Thank you for feedback . I will change it now .

File: javascript/javascript1/week1/Goodboy-Oldboy.js (hunk 5 above)
Comment (suvarna-ratna) [reply javascript/javascript1/week1/Goodboy-Oldboy.js]: okay kavya thank you.

**Pull request merged**